integration/app/test-demo/server.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable */
+const compression = require('compression');
+const fs = require('fs');
+const url = require('url');
+const { v4: uuidv4 } = require('uuid');
+const AWS = require('aws-sdk');
+/* eslint-enable */
+
+let hostname = '127.0.0.1';
+let port = 8080;
+let protocol = 'http';
+let options = {};
+
+const chime = new AWS.Chime({ region: 'us-east-1' });
+const alternateEndpoint = process.env.ENDPOINT;
+if (alternateEndpoint) {
```

Comment:
We do not need this right now so maybe we can remove it?

Reply:
Removed